superset-frontend/src/components/FaveStar.tsx

```diff
@@ -46,7 +46,12 @@ export default class FaveStar extends React.PureComponent<FaveStarProps> {
           label="fave-unfave"
           tooltip={t('Click to favorite/unfavorite')}
         >
-          <a href="#" onClick={this.onClick} className="fave-unfave-icon">
+          <a
+            href="#"
+            onClick={this.onClick.bind(this)}
```

Comment:
this is a very strange place to see a `bind` - if this needs to happen, pease do it in the constructor.

Reply:
this looks like a bad rebase. I removed these binds in https://github.com/apache/incubator-superset/pull/11101

Reply:
yes - bad rebase